src/central-submissions.rst

```diff
@@ -143,3 +147,52 @@ To connect with Excel or Power BI, follow these steps.
 
+.. tip::
+  If you are having trouble getting Power BI to connect to Central, and especially if you see error messages about permissions or authentication, try `resetting the Power BI cached credentials <https://community.powerbi.com/t5/Power-Query/Power-BI-Web-cached-credentials-For-OData/m-p/126826/highlight/true#M8228>`_.
+
+You can also access the OData feed yourself. The OData feed is an easily consumable JSON data format and offers a metadata schema, some filtering and paging options, and more. To learn more about the OData feed, click the :guilabel:`API Access` button or see the `developer documentation <https://odkcentral.docs.apiary.io/#reference/odata-endpoints>`_ directly.
+
+.. _central-submissions-review-states:
+
+Submission Review States
+------------------------
+
+As of version 1.2, Central allows Project Managers and Administrators to review submissions and assign them certain states. This optional feature lets you perform verification and data correction within Central itself. The available states are:
+
++------------+-------------+-----------------------------------------------------------------------------------+
+| State      | Assigned by | Description                                                                       |
++============+=============+===================================================================================+
+| Received   | System      | The default state for all incoming submissions, assigned automatically by Central |
++------------+-------------+-----------------------------------------------------------------------------------+
+| Edited     | System      | Automatically assigned by Central whenever a submission is edited by any user     |
++------------+-------------+-----------------------------------------------------------------------------------+
+| Has Issues | User        | Can be assigned by project staff if a submission has problems                     |
++------------+-------------+-----------------------------------------------------------------------------------+
+| Approved   | User        | Can be assigned by project staff to approve a submission                          |
++------------+-------------+-----------------------------------------------------------------------------------+
+| Rejected   | User        | Can be assigned by project staff to reject a submission                           |
++------------+-------------+-----------------------------------------------------------------------------------+
+
+The ``Received`` and ``Edited`` states are automatically set by Central any time a submission is uploaded or edited. The other states are assigned by project staff. Review states have suggested meanings, but *they do not do anything by themselves*. You are free to use these states however you'd like.
```

Comment:
I probably wouldn't emphasize they don't do anything by themselves.

Reply:
i'd like to get @lognaturel's thoughts on this one?
i feel like i've written these docs with positioning we'd discussed previously. but maybe i'm wrong.

Reply:
On first reading, I was confused about what it would mean for the states to do something by themselves (like are they going to go have a beer together?!). Maybe something like 

> Review states have suggested meanings but they don't trigger automatic behavior. For example, rejected submissions are included in exports unless you explicitly filter them out. You are free to use these states however you'd like or to ignore them entirely.